cumulusci/tests/test_integration_infrastructure.py

```diff
@@ -3,25 +3,119 @@
 import pytest
 
 from cumulusci.tasks.preflight.packages import GetInstalledPackages
+from cumulusci.tasks.salesforce import SOQLQuery
+
+pytestmark = pytest.mark.random_order(disabled=True)
```

Comment:
After reading, I see why these tests are order-dependent, but a note might be helpful.